docs/gallery.yml

```diff
@@ -0,0 +1,57 @@
+  repository: https://github.com/jupyter-book/workshop-template
+  image: https://github.com/jupyter-book/workshop-template/blob/main/content/figures/logo.svg
+  caption:
+  description: A GitHub Template repository designed for use in Jupyter Book 2 and MyST workshops.
+
+- name: The Turing Way handbook to reproducible, ethical and collaborative data science.
+  website: https://book.the-turing-way.org/
+  repository: https://github.com/the-turing-way/the-turing-way
+  image: https://github.com/the-turing-way/the-turing-way/blob/main/book/figures/theturingway-pathway.jpg
+  caption: 
+  description: The Turing Way is an open science, open collaboration, and community-driven project. We involve and support a diverse community of contributors to make data science accessible, comprehensible and effective for everyone.
+
+- name: Myst Official Documentation
+  website: https://mystmd.org/guide
+  repository: https://github.com/jupyter-book/mystmd
+  image: https://github.com/jupyter-book/mystmd/blob/main/docs/videos/links.mp4
+  caption: Live graphs can be embedded directly in your documentation or articles with computation backed by Jupyter or JupyterLite – running locally, on Binder, or directly in your browser.
+  description: MyST is an ecosystem of open-source, community-driven tools designed to revolutionize scientific communication. Our powerful authoring framework supports blogs, online books, scientific papers, reports and journals articles.
+
+- name: Introducing Classical Mechanics & Special Relativity
+  website: https://freekpols.github.io/Mechanica/
+  repository: https://github.com/freekpols/Mechanica
+  image: https://freekpols.github.io/Mechanica/build/cover-f2651612bb78120d7ad964dc4d5f8d4b.gif
+  caption: 
+  description: This book provides an introduction for freshman students into the world of classical mechanics and special relativity theory
+
+- name: Project Pythia
+  website: https://projectpythia.org/
+  repository: https://github.com/ProjectPythia
+  image: https://raw.githubusercontent.com/ProjectPythia/pythia-config/main/logos/pythia_logo-white-rtext-blue.svg
+  caption: 
+  description: Project Pythia is the education working group for Pangeo and is an educational resource for the entire geoscience community. Together these initiatives are helping geoscientists make sense of huge volumes of numerical scientific data using tools that facilitate open, reproducible science, and building a community of practice around these goals.
+
+- name: IRSA Tutorials
+  website: https://caltech-ipac.github.io/irsa-tutorials/
+  repository: https://github.com/Caltech-IPAC/irsa-tutorials/
+  image: https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRVO9k5CiRBcwjV4SozhB2eVY3g21aYfcvSaA&s
+  caption: IRSA Tutorials logo
+  description: IRSA (Infrared Science Archive) provides a variety of tutorials to help users understand and utilize the data and tools available through the archive.
```

Comment:
Do we need to have a different caption and description? Couldn't the description function as the caption?

Reply:
Based on the popularity of the JB1 gallery, I would in fact vote for having only one tile for each book rather than two. If we have any such scripting available it could of course flip from the image to the description on hover, but that would be just cherry on top in fanciness.